src/libraries/Microsoft.PowerFx.Core/Binding/Binder.cs

```diff
@@ -41,6 +41,8 @@ internal sealed partial class TexlBinding
         // Name of entity where this formula was bound.
         public readonly DName EntityName;
 
+        public readonly DName CurrentPropertyName;
```

Comment:
>CurrentPropertyName [](http://example.com/codeflow?start=30&length=19)

Why do we need this?

Reply:
This might be removed eventually, I added this to discuss with the team to see what other ways we have to solve the same issue.